jnigen/tool/pr_checks.dart

```diff
@@ -16,26 +16,47 @@
 import 'dart:async';
 import 'dart:io';
 
+import 'package:args/args.dart';
+
 import 'command_runner.dart';
 
-void main() async {
+// Flags
+const _clone = 'clone';
+
+void main(List<String> arguments) async {
+  final parser = ArgParser()
+    ..addFlag(
+      _clone,
+      help:
+          'Git clone will be run and the checks will happen in a temporary directory.',
```

Comment:
Nit: I think it's conventional to write an imperative statement here.

`Run checks in a cloned copy of the project.`

Reply:
Much clearer than my statement! Will change it.